docs/build-haiku.md

```diff
@@ -0,0 +1,44 @@
+Haiku builds can be created using Clang or GCC using Meson buildsystem.
```

Comment:
(with adjective)

Haiku builds can be created using Clang or GCC using **the** Meson buildsystem.

or (by name):

Haiku builds can be created using Clang or GCC using Meson.

You pick :-)

Reply:
(I prefer the 1st, and have used that in the macOS feedback)